packages/heartwood-components/components/message/message.hbs

```diff
@@ -0,0 +1,31 @@
+<div class="message">
+    <span class="message__pre">
+        {{#if from.image}}
+        {{#if from.isSprucebot}}
+        <img class="message__from-image{{#if isSprucebot}} message__from-sprucebot-image{{/if}}" src="{{ from.image.src }}"
```

Comment:
@jtini looks like the `message__from-sprucebot-image` class isn't being applied

Reply:
fixed